packages/@okta/i18n/src/properties/login_zh_CN.properties

```diff
@@ -335,24 +335,24 @@ enroll.choices.setup.skip = \u8DF3\u8FC7\u8BBE\u7F6E
 enroll.securityQuestion.setup = \u8BBE\u7F6E\u5BC6\u7801\u63D0\u793A\u95EE\u9898\u9A8C\u8BC1
 # Security Questions
 security.disliked_food = \u60A8\u4ECE\u5C0F\u6700\u4E0D\u559C\u6B22\u7684\u98DF\u7269\u662F\u4EC0\u4E48?
-security.name_of_first_plush_toy = \u60A8\u7684\u7B2C\u4E00\u4E2A\u6BDB\u7ED2\u73A9\u5177\u53EB\u4EC0\u4E48\u540D\u5B57?
-security.first_award = \u60A8\u83B7\u5F97\u7B2C\u4E00\u4E2A\u5956\u7AE0\u6216\u5956\u54C1\u7684\u662F\u4EC0\u4E48\u6D3B\u52A8?
-security.favorite_security_question = \u60A8\u6700\u559C\u6B22\u7684\u5B89\u5168\u95EE\u9898\u662F\u4EC0\u4E48?
-security.favorite_toy = \u60A8\u4ECE\u5C0F\u6700\u559C\u6B22\u7684\u73A9\u5177/\u6BDB\u7ED2\u73A9\u5177\u662F\u4EC0\u4E48?
-security.first_computer_game = \u60A8\u73A9\u8FC7\u7684\u7B2C\u4E00\u6B3E\u8BA1\u7B97\u673A\u6E38\u620F\u662F\u4EC0\u4E48?
-security.favorite_movie_quote = \u60A8\u6700\u559C\u6B22\u7684\u7535\u5F71\u53F0\u8BCD\u662F\u4EC0\u4E48?
-security.first_sports_team_mascot = \u60A8\u53C2\u52A0\u8FC7\u7684\u7B2C\u4E00\u4E2A\u8FD0\u52A8\u961F\u7684\u5409\u7965\u7269\u662F\u4EC0\u4E48?
-security.first_music_purchase = \u60A8\u7B2C\u4E00\u6B21\u8D2D\u4E70\u7684\u97F3\u4E50\u4E13\u8F91\u6216\u6B4C\u66F2\u662F\u4EC0\u4E48?
-security.favorite_art_piece = \u60A8\u6700\u559C\u6B22\u7684\u827A\u672F\u54C1\u662F\u4EC0\u4E48?
-security.grandmother_favorite_desert = \u60A8\u5976\u5976\u6700\u559C\u6B22\u7684\u751C\u70B9\u662F\u4EC0\u4E48?
-security.first_thing_cooked = \u60A8\u5B66\u4F1A\u70F9\u996A\u7684\u7B2C\u4E00\u79CD\u98DF\u7269\u662F\u4EC0\u4E48?
-security.childhood_dream_job = \u60A8\u4ECE\u5C0F\u6700\u68A6\u60F3\u7684\u5DE5\u4F5C\u662F\u4EC0\u4E48?
-security.first_kiss_location = \u60A8\u7684\u521D\u543B\u53D1\u751F\u5728\u4EC0\u4E48\u5730\u65B9?
-security.place_where_significant_other_was_met = \u60A8\u9047\u5230\u60A8\u7684\u914D\u5076/\u91CD\u8981\u7684\u53E6\u4E00\u534A\u662F\u5728\u4EC0\u4E48\u5730\u65B9?
-security.favorite_vacation_location = \u60A8\u6700\u559C\u6B22\u53BB\u54EA\u91CC\u5EA6\u5047?
-security.new_years_two_thousand = 2000 \u5E74\u9664\u5915\u591C\u60A8\u5728\u4EC0\u4E48\u5730\u65B9?
-security.favorite_speaker_actor = \u60A8\u6700\u559C\u6B22\u7684\u6F14\u8BB2\u8005/\u6F14\u8BB2\u5BB6\u662F\u8C01?
-security.favorite_book_movie_character = \u60A8\u6700\u559C\u6B22\u7684\u4E66\u672C/\u7535\u5F71\u4EBA\u7269\u662F\u8C01?
+security.name_of_first_plush_toy = \u60A8\u7684\u7B2C\u4E00\u4E2A\u6BDB\u7ED2\u73A9\u5177\u53EB\u4EC0\u4E48\u540D\u5B57\uFF1F
```

Comment:
Looks like we removed the trailing `?` from a bunch of these keys. Its still present for a few of them 
```
security.disliked_food = \u60A8\u4ECE\u5C0F\u6700\u4E0D\u559C\u6B22\u7684\u98DF\u7269\u662F\u4EC0\u4E48? 

security.favorite_sports_player = \u60A8\u6700\u559C\u6B22\u7684\u8FD0\u52A8\u5458\u662F\u8C01?

``` 
Should they be removed ?

Reply:
@nikhilvenkatraman-okta The `?` was not removed. We are using the ASCII question mark, which is incorrect. It's now replaced with the full-width Chinese punctuation mark \uFF1F instead. See: https://unicode-table.com/en/FF1F/